spec/controllers/v2/security_guides_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe V2::SecurityGuidesController do
+  let(:security_guides) { FactoryBot.create_list(:v2_security_guide, 20) }
+
+  context 'Unauthorized' do
+  end
+
+  context 'Authorized' do
+    let(:current_user) { FactoryBot.create(:v2_user) }
+
+    before(:each) do
+      request.headers['X-RH-IDENTITY'] = current_user.account.identity_header.raw
+      expect(controller).to receive(:rbac_allowed?).and_return(rbac_allowed?)
+    end
+
+    describe 'GET index' do
+      context 'RBAC allowed' do
+        let(:rbac_allowed?) { true }
+
+        it 'responds with ok status' do
+          get :index
+
+          expect(response).to have_http_status :ok
+        end
+
+        it 'returns security guides' do
+          sg = security_guides.first
+
+          get :index
+
+          expected = {
+            'id': sg.id,
+            'ref_id': sg.ref_id,
+            'version': sg.version
+          }
+
+          expect(response.parsed_body).to include(expected)
```

Comment:
You can merge this and the previous test together, meaning the response expectation can be moved here and the previous one can be deleted. Then the test itself talks about security guides in plural but you are just testing the existence of a single entity in it, not even the fact if it is an array or not. Furthermore, you can write some smart iteration to check all 3 field in a one-liner if really necessary...